### PR TITLE
Add donor criteria field

### PR DESCRIPTION
**Pull request by dinovski**

### CODE REVIEW

File: bhot/transplants/models/transplant.py [hunk 1]
```diff
@@ -17,25 +18,480 @@ class Sex(models.TextChoices):
...
+    )
+    recipient_diastolic_bp = models.IntegerField(
+        blank=True,
+        null=True,
+        verbose_name="recipient diastolic blood pressure (mm Hg)",
+    )
+    primary_kidney_disease = models.CharField(
+        max_length=200,
+        blank=True,
+        null=True,
```
Comment (wapiflapi) [inline bhot/transplants/models/transplant.py:457]: CharField with null

**Review by wapiflapi — CHANGES_REQUESTED** [review]

File: bhot/transplants/models/sequencing.py [hunk 2]
```diff
@@ -1,21 +1,52 @@
...
         verbose_name_plural = "sequencing data"
 
+    class RunProtocol(models.TextChoices):
+        SENSITIVE = "high sensitivity", _("high sensitivity")
+        STANDARD = "standard", _("standard")
+
     id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
     biopsy = models.ForeignKey(Biopsy, null=True, on_delete=models.SET_NULL)
-    sequencing_date = models.DateField()
+    transplant = models.ForeignKey(Transplant, null=True, on_delete=models.SET_NULL)
```
[the remark this answers is not in the document]
Comment (wapiflapi) [reply bhot/transplants/models/sequencing.py:23]: We will be using the `run_date` instead. We will have to update this where identity is concerned.

File: bhot/transplants/models/biopsy.py [hunk 3]
```diff
@@ -10,6 +12,332 @@ class Biopsy(UserScopedModel):
...
+        blank=True,
+        choices=Immunosuppressants.choices,
+        verbose_name="Immunosuppresants",
+    )
+    immunosuppressant_dose = models.FloatField(
+        blank=True,
+        null=True,
+    )
+    immunosuppressant_dose_units = models.CharField(
+        null=True,
```
[the remark this answers is not in the document]
Comment (dinovski) [reply bhot/transplants/models/biopsy.py:206]: units are only required if there is an entry for dose so I will leave it as is for now.

File: bhot/transplants/models/histology.py [hunk 4]
```diff
@@ -10,6 +12,638 @@ class Histology(UserScopedModel):
...
+        NOS = "not otherwise specified", _("not otherwise specified (NOS)")
+        PERIHILIAR = "perihiliar", _("perihiliar")
+        TIP = "tip", _("tip")
+
+    class BiopsyQuality(models.TextChoices):
+        ADEQUATE = "adequate", _("adequate")
+        MINIMAL = "minimal", _("minimal")
+
+    class TMAlocation(models.TextChoices):
+        ADEQUATE = "arteriolar", _("arteriolar")
```
[the remark this answers is not in the document]
Comment (dinovski) [reply bhot/transplants/models/histology.py:42]: nope :)

File: bhot/transplants/models/sequencing.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (dinovski) [reply bhot/transplants/models/sequencing.py:23]: removed transplant, it was a copy/paste error.

**Pull request merged**